src/components/CalloutBanner.tsx

```diff
@@ -1,77 +1,70 @@
-import { useTranslation } from "next-i18next"
-import { Flex, type FlexProps } from "@chakra-ui/react"
-
-import type { TranslationKey } from "@/lib/types"
+import { Box, Flex, type FlexProps, Heading, Text } from "@chakra-ui/react"
 
 import { Image, type ImageProps } from "@/components/Image"
-import OldHeading from "@/components/OldHeading"
-import Text from "@/components/OldText"
 
 export type CalloutBannerProps = FlexProps & {
   children?: React.ReactNode
   image: ImageProps["src"]
   imageWidth?: number
-  titleKey: TranslationKey
-  descriptionKey: TranslationKey
+  title: string
+  description: string
   alt: string
+  alignImage?: FlexProps["alignItems"]
 }
 
 const CalloutBanner = ({
   image,
   imageWidth,
-  titleKey,
-  descriptionKey,
+  title,
+  description,
   alt,
   children,
+  alignImage = "center",
   ...props
 }: CalloutBannerProps) => {
-  const { t } = useTranslation("page-staking")
-
   return (
     <Flex
       as="aside"
-      direction={{ base: "column", lg: "row-reverse" }}
+      flexDir={{ base: "column", md: "row-reverse" }}
       bg="layer2Gradient"
-      p={{ base: 8, sm: 12 }}
       borderRadius="base"
       {...props}
     >
       {image && (
-        <Flex>
+        <Flex
+          flex="1 1 50%"
+          position="relative"
+          justify="center"
+          align={alignImage}
+          minH={{ base: 200, md: "auto" }}
+          px={{ base: 8, md: 0 }}
+        >
           <Image
             src={image}
-            alt={alt}
             width={imageWidth}
+            alt={alt}
+            position="absolute"
+            w="full"
+            maxH="150%"
```

Comment:
We used 120% above, just double checking this.

<img width="1800" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/f3185174-813c-4833-ba04-0bce93746da2">